class/Admin/PostData.php

```diff
@@ -186,6 +186,8 @@ public function save_notification_data( $data ) {
 
 	        }
 
+	        $notification_data = apply_filters( 'notification/allow_unfiltered_body', $notification_data, $ndata );
```

Comment:
I think this name should me more general, something like `notification/notification/form/data/values`